docs/releases/v8_15.md

```diff
@@ -0,0 +1,36 @@
+# Release Notes
+
+## March 2023 (version 8.15)
+
+### Overview
+
+The **March 11th, 2023** release of **DietPi v8.15** comes with new images for the StarFive **VisionFive 2 RISC-V board**, the PINE64 **Quartz64** and a couple of improvements and bug fixes.
+
+![Primrose](../assets/images/dietpi-release-v8_15.jpg){: width="480" height="320" loading="lazy"}
+
+!!! cite "*Photo by `Alicja`, Pixabay*"
+
+### New images
+
+- **StarFive VisionFive 2** :octicons-arrow-right-16: An experimental image has been added. For details, please read our related blog post: <https://dietpi.com/blog/?p=2629>
+- [**Quartz64**](../../hardware/#pine64) :octicons-arrow-right-16: Major kernel update to latest v6.1.14, stable U-Boot v2022.04 and enabled a lot of kernel features for various container engines and firewall/filter tasks: <https://github.com/MichaIng/DietPi/issues/6151>
```

Comment:
Not sure if this fits to "New images", since the images generally existed before and the new kernel is applied on DietPi update to old images as well. I fear this causes the impression that one needs to flash a new image. The hint with the update should be added (will do on DietPi CHANGELOG.txt as well):
```suggestion
- [**Quartz64**](../../hardware/#pine64) :octicons-arrow-right-16: Major kernel update to latest v6.1.14, stable U-Boot v2022.04 and enabled a lot of kernel features for various container engines and firewall/filter tasks: <https://github.com/MichaIng/DietPi/issues/6151>. The kernel upgrade will be applied on DietPi update.
```